src/native/minipal/utf8.h

```diff
@@ -68,6 +68,22 @@ size_t minipal_convert_utf8_to_utf16(const char* source, size_t sourceLength, CH
  */
 size_t minipal_convert_utf16_to_utf8(const CHAR16_T* source, size_t sourceLength, char* destination, size_t destinationLength, unsigned int flags);
 
+/**
+ * Convert a UTF-16 character to uppercase using invariant culture.
+ *
+ * @param code The UTF-16 character to be converted.
+ * @return The uppercase equivalent of the character or the character itself if no conversion is necessary.
+ */
+const CHAR16_T minipal_toupper_invariant(const CHAR16_T code);
```

Comment:
```suggestion
CHAR16_T minipal_toupper_invariant(CHAR16_T code);
```
Does `const` serve any purpose here?

Reply:
Could the input stay `const` (because it's not changing in the scope of this function)?

Reply:
It is unnecessary clutter in the public signature.

Reply:
Removed. (I saw it in existing signatures in this file but those are pointers).